pkgs/default.nix

```diff
@@ -35,6 +35,8 @@ let
         pkgs = pkgs.lib.mkForce pkgs; # to override ./modules/nixpkgs/config.nix
       };
 
+      system.stateVersion = "22.11";
```

Comment:
Hard coding a state version inside here forces users to use mkForce which doesn't be required. I would recommend to either use lib.mkDefault or better omit it and force users to set it themselves.

Reply:
This change only affects the bootstrap zip generation. For this we create a nix-on-droid configuration. On device, this file here is not used anymore.